src/HotChocolate/Core/src/Types/Types/Descriptors/Conventions/DefaultNamingConventions.cs

```diff
@@ -144,7 +143,7 @@ public virtual NameString GetTypeName(Type type, TypeKind kind)
 
             string name = type.GetGraphQLName();
 
-            if (kind == TypeKind.InputObject
+            if ((kind == TypeKind.InputObject || kind == TypeKind.InputUnion)
```

Comment:
I think this is not a good idea. For input objects the name extension is a good practice but for unions? There is nothing like that established yet. Or is there?

Reply:
Looks like it is also like this in the spec